app/views/includes/profile-header.nunjucks

```diff
@@ -9,7 +9,8 @@
       {% if gp.contact.telephone %}Reception: {{ gp.contact.telephone }}<br />{% endif %}
       {% if gp.contact.fax %}Fax: {{ gp.contact.fax }}<br />{% endif %}
       {% if gp.contact.email %}<a href="mailto:{{ gp.contact.email }}">{{ gp.contact.email }}</a><br />{% endif %}
-
+      {% if gp.contact.website %}<a href="http://{{ gp.contact.website }}">{{ gp.contact.website }}</a><br />{% endif %}
```

Comment:
Due to variable data, prepending `http://` isn't going to work for everything e.g. http://profiles-pr-101.dev.c2s.nhschoices.net/gp-surgeries/36798. Suggest a helper function to add the protocol if not already present.